presto-main/src/main/java/com/facebook/presto/server/TaskInfoResource.java

```diff
@@ -69,4 +118,34 @@ public TaskInfo getTaskInfo(@PathParam("taskId") TaskId taskId)
             throw new NotFoundException(e);
         }
     }
+
+    private void proxyTaskInfo(HttpServletRequest servletRequest, AsyncResponse asyncResponse, String xForwardedProto, UriInfo uriInfo)
```

Comment:
I think xForwardedProto is not used anywhere

Reply:
Removed `xForwardedProto`